test/benchmark/config/nighthawk-test-server.yaml

```diff
@@ -0,0 +1,53 @@
+### Nighthawk test server deployment & service
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nighthawk-test-server
+  namespace: benchmark-test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nighthawk-test-server
+  template:
+    metadata:
+      labels:
+        app: nighthawk-test-server
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: nighthawk-server
```

Comment:
can replace this test server with a much simpler one, like echo server. can be done as a follow-up PR.